### PR TITLE
Add datetime label to logging

### PR DESCRIPTION
**Pull request by vikingUnet**
Sometimes it is usefull to see datetime of starting each launch and each operation in logfile.

### CODE REVIEW

File: ldap2pg/config.py [hunk 1]
```diff
@@ -636,11 +637,11 @@ def logging_dict(self):
             'formatters': {
                 'info': {
                     '()': __name__ + '.MultilineFormatter',
-                    'format': '%(message)s',
+                    'format': '%(asctime)s %(levelname)5.5s: %(message)s',
```
Comment (bersace) [inline ldap2pg/config.py:640]: I suggest double spaces avec colon, like PostgreSQL:

```diff
-%(levelname)5.5s: %(message)s
+%(levelname)s:  %(message)s
```

File: ldap2pg/config.py [hunk 2]
```diff
@@ -538,7 +539,7 @@ def load(self, argv=None):
             self['color'] = args.color
         dictConfig(self.logging_dict())
 
-        logger.info("Starting ldap2pg %s.", __version__)
+        logger.info("Starting ldap2pg %s at %s.", __version__, datetime.now().isoformat(sep=' ', timespec='seconds'))
```
Comment (bersace) [inline ldap2pg/config.py:542]: Good ! You should split this long line.

**Review by bersace — COMMENTED** [review]
Thanks for the patch !

Could you add a changelog entry in CHANGELOG.md ?

**Review by bersace — APPROVED** [review]
Thanks !

**Comment by vikingUnet** [comment]
Thanks to you!

**Pull request merged**